jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/layout/GridListTests.java

```diff
@@ -0,0 +1,18 @@
+package io.github.epam.material.tests.layout;
+
+import static io.github.com.StaticSite.gridListFrame;
+import static io.github.com.StaticSite.layoutGridListPage;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.Test;
+
+public class GridListTests extends TestsInit {
+
+    @Test
+    public void gridListTest() {
```

Comment:
Can we check that we can get section by index and check that found section is displayed